test/unit/spec/narrative_core/jobCommChannel-spec.js

```diff
@@ -132,20 +132,18 @@ define(['jobCommChannel', 'base/js/namespace', 'common/runtime', 'testUtil'], (
             },
             {
                 channel: 'request-job-status',
-                message: { jobId: 'someJob', parentJobId: 'someParent' },
```

Comment:
I removed all the references to `parentJobId` in these tests as the backend no longer takes `parent_job_id` as a parameter for these queries.